src/batch.ts

```diff
@@ -170,13 +186,21 @@ export interface BatchError {
  * @param tEnd End of the integration (must be greater than `tStart`)
  *
  * @param control Optional control parameters to tune the integration
+ *
+ * @param immediate Indicates if we should immediately compute all
+ * solutions in the batch (may take a while).
  */
 export function batchRun(Model: OdinModelConstructable, pars: BatchPars,
                          tStart: number, tEnd: number,
-                         control: Partial<DopriControlParam>): Batch {
+                         control: Partial<DopriControlParam>,
+                         immediate: boolean = true): Batch {
     const run = (p: UserType, t0: number, t1: number) =>
         wodinRun(Model, p, t0, t1, control);
-    return new Batch(run, pars, tStart, tEnd);
+    const ret = new Batch(run, pars, tStart, tEnd);
+    if (immediate) {
+        ret.run();
+    }
```

Comment:
Is this just to reproduce the current behaviour where blocking run happens immediately?

Reply:
yes - if we change the ODE batch version later, we can remove this option